apps/condo/domains/organization/integrations/sbbol/sync/syncBankAccounts.js

```diff
@@ -63,7 +64,26 @@ const _syncBankAccounts = async (accounts, organization) => {
                     organization: { connect: { id: organization.id } },
                 }
             )
-            logger.info('Created BankAccount', { bankAccount: { id: account.number, organization: { id: organization.id, name: organization.name } } })
+            logger.info({ msg: 'Created BankAccount', bankAccount: { id: account.number, organization: { id: organization.id, name: organization.name } } })
+        } else {
+            const haveSBBOLIntegrationContext = get(foundAccount, 'integrationContext.integration.id') === integration.id
```

Comment:
In case when `foundAccount` will have other non-SBBOL integration, `haveSBBOLIntegrationContext` will be equal to false, `BankIntegrationContext.create` will be executed, we get an instance, then `BankAccount.update` will be failed and the instance will left to be orphaned.